cspell.yaml

```diff
@@ -51,6 +51,11 @@ words:
 - OBS
 - HttpFS
 - safemode
+- mis-replicate
+- mis-replication
+- mis-replicated
```

Comment:
I think `misreplication` is [one word](https://en.wiktionary.org/wiki/misreplication), similar to [other words](https://en.wiktionary.org/wiki/Category:English_terms_prefixed_with_mis-) with `mis-` prefix.

CC @sodonnel

Reply:
Seems like it should be, although I think we have used mis-replicated in some headings in the source code. Eg the RM report has Mis-Replicated containers. I think we may have carried the spelling over from HDFS which I think spells it the same way.

Reply:
I actually saw the same wikipedia page when I was looking this up. The point of these configs is to make sure we use consistent spelling and capitalization of terms throughout the docs, so this is where we can decide whether we want to use `misreplication` or `mis-replication` across the whole site. cspell also doesn't recognize `misreplication` so we can choose one to add and the other will be flagged as incorrect.

Reply:
As for which one to use, I'm not sure I have a preference. Misreplication looks more correct, although it would be used with overreplication and underreplication which look a little strange. As Stephen said, I'm pretty sure code and CLI output uses mis-replication, over-replication, and under-replication, so maybe it would be easiest just to keep using those?

Reply:
Saying as the code has mis-replication, I am happy to go with that. Otherwise we are going to have to make code changes just to facilitate the docs.

Reply:
Ok let's keep "mis-replication". "overreplication" and "underreplication" will already be flagged. CSpell doesn't support matching on spaces so we can't forbid "over replication" in favor of "over-replication" with cspell, although I could just add a `grep` at the end of the `spelling.sh` file that does this.